src/Packetery/Module/Dashboard/DashboardHelper.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Packetery\Module\Dashboard;
+
+use Packetery\Module\Shipping\ShippingProvider;
+use WC_Data_Store;
+use WC_Shipping_Zone;
+use WC_Shipping_Zone_Data_Store;
+
+class DashboardHelper {
+	/**
+	 * Tells if there is Packeta shipping method configured and active.
+	 *
+	 * @return bool
+	 */
+	public function isPacketaShippingMethodActive(): bool {
+		/** @var WC_Shipping_Zone_Data_Store $shippingDataStore */
+		/** @phpstan-ignore varTag.type */
+		$shippingDataStore = WC_Data_Store::load( 'shipping-zone' );
```

Comment:
Nechceme to volání statické metody z WC přesunout taky do Frameworku  ?  Už to máme na druhém místě.